packages/blade/src/components/Badges/_decisions/decisions.md

```diff
@@ -0,0 +1,61 @@
+# Badge <!-- omit in toc -->
+Badges are used to show small amount of color coded metadata, which are ideal for getting users attention.
+
+<img src="./badge-thumbnail.png" width="380" />
+
+## Design
+- [Figma - Badge](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=8110%3A417)
+
+
+## API
+
+| Prop | Type | Required | Default | Description |
+|---|---|---|---|---|
+| variant | `positive`,`negative`, `notice`, `information`, `neutral` | No | `neutral` | The variant of the badge to be rendered. |
```

Comment:
```suggestion
| variant | `positive`,`negative`, `notice`, `info`, `neutral` | No | `neutral` | The variant of the badge to be rendered. |
```

can we make this `info`? Alert has `info`

Reply:
Our tokens have `information`. BaseButton & Link `intent` has `information`. We should keep it similar everywhere. @divyanshu013 should we make it `information` for Alert?

Reply:
@saurabhdaware Divyanshu changed it to `information` on Alert. Lets stick to `information` since the convention would be similar across components and design